Client/Assets/CC_Script/Helpers.ios.mjs

```diff
@@ -42,6 +42,12 @@ HTMLElement.prototype.getAutocompleteInfo = function() {
   };
 };
 
+// Bug 1835024. Webkit doesn't support `checkVisibility` API
+// https://drafts.csswg.org/cssom-view-1/#dom-element-checkvisibility
+HTMLElement.prototype.checkVisibility = function (options) {
+  throw new Error(`Not implemented: WebKit doesn't support checkVisibility `);
+};
+
```

Comment:
@nbhasin2 this method will be really useful for the field navigation for detecting invisible inputs ( once it gets implemented in webkit 😄 )